src/components/FeatureViews/Accounts/ResetPasswordDialogs.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+
+import { ResponsiveButton } from "../../../theme/ThemedComponents/Button/ResponsiveButton";
+import { fullName } from "utils";
+
+
+const ResetPasswordDialogs = ({userInfo, open, openReset, password, handleClosePassword, handleClosePasswordReset, handleCloseReset}) => {
+
+    return (
+    <>
+      <Dialog
+        open={open}
+        onClose={handleClosePassword}
+        maxWidth="xs"
+        aria-labelledby="alert-dialog-title"
```

Comment:
to help the reader - can you rename this aria-labelledby to "step1-confirm-reset-user-password-alert" ?

Reply:
Done!